mautic/mautic
# change check to array, fix existing themes

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Check the list of available themes. Among others "Blank" should be available. These themes should **not** be available:
  - Brienz
  - Confirm Me
  - Paprika
3. Enable grapesjs
4. The 3 following 3 themes should now be available in addition to "Blank". 
  - Brienz
  - Confirm Me
  - Paprika

fixes: https://github.com/mautic/plugin-grapesjs-builder/issues/51
